documentation/recommendations.adoc

```diff
@@ -799,24 +797,89 @@ RETURN m.title, SUM( pearson * r.rating) AS score
 ORDER BY score DESC LIMIT 25
 ----
 
+++++
+</div>
+++++
+
+== Collaborative Filtering – Neighborhood-Based Recommendations
+
 
-.kNN movie recommendation using Pearson similarity function
+++++
+<div class="col-lg-3 ng-scope">
+++++
+
+=== Calculating KNN with GDS
+
+We will now explore how to calculate KNN using GDS.  In order to do this, we will use the following steps:
+
+1. Create a graph projection utilizing the `User` and `Movie` nodes with the `RATED` relationship.
+2. Generate graph embeddings using FastRP.
+3. Calculate KNN.
+
+
+++++
+</div>
+++++
+
+++++
+<div class="col-lg-9 ng-scope">
+++++
+
+
+.Create a graph projection
 [source, cypher]
 ----
-MATCH (u1:User {name: 'Cynthia Freeman'})-[x:RATED]->(movie:Movie)
-WITH u1, gds.alpha.similarity.asVector(movie, x.rating) AS u1Vector
-MATCH (u2:User)-[x2:RATED]->(movie:Movie) WHERE u2 <> u1
+CALL gds.graph.project(
+	'ratings',
+    ['User', 'Movie'],
+    {
+    	RATED: {
+        	orientation: 'UNDIRECTED',
+            properties: 'rating'
+        }
+    }
+)
+----
 
-WITH u1, u2, u1Vector, gds.alpha.similarity.asVector(movie, x2.rating) AS u2Vector
-WHERE size(apoc.coll.intersection([v in u1Vector | v.category], [v in u2Vector | v.category])) > 10
+.Generate FastRP embeddings
```

Comment:
shouldn't this just be nodesimilarity with rating as relationshipWeight? I think FastRP is unneccesary and overkill here (esp. also as this is not a GDS guide/sandbox) but just a plain jaccard sim.

Reply:
This has been discussed in [this Slack thread](https://neo4j.slack.com/archives/C5DABHVS4/p1649346467106019).  Ultimately we decided to (temporarily) remove this query from the Sandbox until the team can identify proper strategies and messaging around FastRP+KNN for recommendations.  The section has been commented out as per the request of @jexp so it can always be added in later.